workshop_app/templates/workshop_app/activation.html

```diff
@@ -1,23 +1,25 @@
 {% extends 'workshop_app/base.html' %}
 
 {% block title %}
-	Awating activation
+    Awaiting activation
 {% endblock %}
 
 {% block header %}
-	<nav class="navbar navbar-default navbar-custom">
-	  <div class="container-fluid">
-		<div class="navbar-header">
-		  <a class="navbar-brand" href="#">FOSSEE Python Workshops</a>
-		</div>
-		<ul class="nav navbar-nav navbar-right">
-		{% if request.user.profile.is_email_verified %}
-		  <li><a href="{{ URL_ROOT }}/view_profile/"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
-		
-		  <li><a href="{{ URL_ROOT }}/logout/"><span class="glyphicon glyphicon-log-out"></span> Logout</a></li>
-		{% endif %}
-		</ul>
-	  	</div>
+    <nav class="navbar navbar-default navbar-custom">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="#">FOSSEE Python Workshops</a>
+            </div>
+            <ul class="nav navbar-nav navbar-right">
+                {% if request.user.profile.is_email_verified %}
+                    <li><a href="{{ URL_ROOT }}/view_profile/"><span class="glyphicon glyphicon-user"></span>
```

Comment:
Wherever `{{ URL_ROOT }}/view_profile/` this format is used, replace that with `{% url '<app_name>:<url_name>' args %}` format.
Make the above change wherever necessary.

Reply:
Will be fixing in a later PR